src/Umbraco.Cms.ManagementApi/Factories/PagedViewModelFactory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Umbraco.Cms.ManagementApi.ViewModels.Pagination;
+
+namespace Umbraco.Cms.ManagementApi.Factories;
+
+public class PagedViewModelFactory : IPagedViewModelFactory
+{
+    public PagedViewModel<T> Create<T>(IEnumerable<T> items, int skip, int take)
```

Comment:
I am not sure I understand the purpose of this..? When do we want to create a paginated view model *after* loading every available item? Doesn't this implementation promote the bad practice of always paginating in memory over the entire result set?